src/EnergyPlus/ConvectionCoefficients.cc

```diff
@@ -3213,41 +3034,36 @@ void SetupAdaptiveConvRadiantSurfaceData(EnergyPlusData &state)
     // and ZoneEquipConfig(ZoneNum)%InFloorActiveElement.
 
     for (int ZoneLoop = 1; ZoneLoop <= state.dataGlobal->NumOfZones; ++ZoneLoop) {
-        state.dataConvectionCoefficient->ActiveWallCount = 0;
-        state.dataConvectionCoefficient->ActiveWallArea = 0.0;
-        state.dataConvectionCoefficient->ActiveCeilingCount = 0;
-        state.dataConvectionCoefficient->ActiveCeilingArea = 0.0;
-        state.dataConvectionCoefficient->ActiveFloorCount = 0;
-        state.dataConvectionCoefficient->ActiveFloorArea = 0.0;
-
-        for (int spaceNum : state.dataHeatBal->Zone(ZoneLoop).spaceIndexes) {
+        int activeWallCount = 0;
```

Comment:
These things were in `state` for some reason, but they are essentially locals. I don't think this is an isolated case.